pkg/rpc/server/server.go

```diff
@@ -1538,6 +1539,29 @@ func (s *Server) getUnclaimedGas(ps request.Params) (interface{}, *response.Erro
 	}, nil
 }
 
+// getCandidates returns the current list of candidates with their active/inactive voting status.
+func (s *Server) getCandidates(_ request.Params) (interface{}, *response.Error) {
+	var validators keys.PublicKeys
+
+	validators, err := s.chain.GetNextBlockValidators()
+	if err != nil {
+		return nil, response.NewRPCError("Can't get next block validators", err.Error())
+	}
+	enrollments, err := s.chain.GetEnrollments()
+	if err != nil {
+		return nil, response.NewRPCError("Can't get enrollments", err.Error())
+	}
+	var res = make([]result.Candidate, 0)
+	for _, v := range enrollments {
+		res = append(res, result.Candidate{
+			PublicKey: *v.Key,
+			Votes:     v.Votes.Int64(),
```

Comment:
Can it be a very big int that doesn't fit into int64? Maybe it's better to keep bigint representation and marshal it into string directly?

Reply:
We only have 100M NEO.